### PR TITLE
beamer: events: convert received bytes to HexBytes

### PR DESCRIPTION
**Pull request by istankovic**
This improves the log output of fill IDs.
~(`fill_id` is typed as `FillId`, which is a newtype of `Hexbytes`, but we get `bytes` instances from `web3`, which means we must convert them)~

We need to convert received `bytes` from `web3` to proper `FillId` which is derived from `HexBytes` to improve repr.

### CODE REVIEW

File: beamer/events.py [hunk 1]
```diff
@@ -177,6 +177,9 @@ def _decode_event(
         kwargs["chain_id"] = chain_id
         kwargs["block_number"] = log_entry["blockNumber"]
         kwargs["tx_hash"] = log_entry["transactionHash"]
+        fill_id = kwargs.get("fill_id")
+        if fill_id is not None:
+            kwargs["fill_id"] = FillId(HexBytes(fill_id))
```
Comment (fredo) [inline beamer/events.py:182]: FillId is not a NewType of HexBytes. It's a NewType of bytes
see https://github.com/beamer-bridge/beamer/blob/c5aa401c4984993084a561492ed36bd14ad3f8d9/beamer/typing.py#L13

Comment (istankovic) [reply beamer/events.py]: You're totally right, this had a couple of iterations and the PR description as well as the content is just wrong.
Pushed another version.

**Comment by fredo** [comment]
Looks good! We should consider to use this for other bytes representations in our code base as well. (Like request hash, fill hash)

**Review by fredo — APPROVED** [review]

**Comment by istankovic** [comment]
Pushed a new version that also does the same for `FillHash` and `RequestHash`.

**Comment by fredo** [comment]
awesome!

**Pull request merged**